.github/workflows/updatecli.yml

```diff
@@ -11,19 +11,34 @@ permissions:
 jobs:
   bump:
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
We could use a folder instead:
- https://github.com/updatecli/updatecli/blob/f2bd316dec2460456a780428b6cab1c8f745bfc3/.github/workflows/updatecli.yaml#L36

Then there is no need for a matrix job.

Though, if one of them failed I don't know how this will be behave

Reply:
I tried it with `diff`

```
Run Summary
===========
Pipeline(s) run:
  * Changed:    1
  * Failed:     0
  * Skipped:    0
  * Succeeded:  2
  * Total:      3
```

looking at the summary, I assume succesful pipeline won't be affected by failing ones.

Reply:
https://github.com/elastic/apm-agent-java/pull/3053/commits/8b4db4c12c3d8d191836ce682938caf7f3e0ee51

Reply:
Thanks for double checking this!